README.md

```diff
@@ -293,14 +293,11 @@ you start these dependencies here, there's no need to also attempt to do so in
 the Lighthouse repository. They are the same resources in both and the second
 one to be started will show exceptions about ports already being allocated:
 
-    docker-compose -f dependencies/docker-compose.yml up -d
+    ./dependencies/up.sh
```

Comment:
You can use the standard commands.  You just need to switch to the `dependencies` directory before you run `docker-compose up -d`.  Using this `.sh` file means you never get an unexpected outcome from running it.  It works out the relative path to the script, goes into that directory to run `docker-compose up -d` for you and then when the script finished, you haven't moved directory in your own shell.  I thought these conveniences might be worth it to avoid having to explain too much in the README and the inevitable confusion when you run the command from the wrong directory and the dependencies don't start.

Reply:
Oh by the way I was going to duplicate these scripts in the lighthouse repo so they both work the same way.  I just can't modify both repos in the same PR 😄

Reply:
I think we should keep this as shown here unless it turns out to be an issue.  This is the cleanest solution I could come up with for now.